### PR TITLE
502 treat700 91 spatial objects

### PR DESCRIPTION
**Pull request by dr0i**
See hbz/nwbib#502.

### CODE REVIEW

File: src/test/resources/jsonld/HT014078228 [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    },
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-38#!",
+      "label" : "lobid Organisation"
+    },
+    "label" : "Webseite der hbz-Ressource HT014078228"
+  },
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/HT014078228#!",
+  "spatial" : [ {
```
Comment (dr0i) [inline src/test/resources/jsonld/HT014078228]: @acka47 you want to have a look at this

File: src/test/resources/jsonld/HT019868557 [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      "gndIdentifier" : "128710195",
+      "label" : "Hiepel, Claudia"
+    },
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/cre",
+      "label" : "Autor/in"
+    }
+  } ],
+  "responsibilityStatement" : [ "Claudia Hiepel" ],
+  "coverage" : [ "91" ],
```
Comment (dr0i) [inline src/test/resources/jsonld/HT019868557]: @acka47 you want to have a look at this - added only "coverage"

File: src/main/java/org/lobid/resources/ElasticsearchIndexer.java [hunk 3]
```diff
@@ -313,12 +316,13 @@ private String enrich(final String index, final String queryField,
 								.must(new MultiMatchQueryBuilder(query.first)
 										.fields(WikidataGeodata2Es.FIELD_BOOST)
 										.type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
-										.operator(Operator.AND))
+										.operator(Operator.AND));
+					if (query.second != "")
```
Comment (fsteeg) [inline src/main/java/org/lobid/resources/ElasticsearchIndexer.java]: I think testing with `!isEmpty()` should be safer here due to the reference comparison done by `!=`.

**Comment by dr0i** [comment]
As always, will be deployed to staging at Wednesday around 9:00 am and automatically be switched at Thursday 9:00 am. You may always switch manually.

**Pull request merged**